reaktive/src/commonTest/kotlin/com/badoo/reaktive/single/UpstreamDownstreamGenericTests.kt

```diff
@@ -41,4 +44,28 @@ interface UpstreamDownstreamGenericTests {
                 }
             }
     }
+
+    object TransformToMaybe {
```

Comment:
It can be just a function inside existing companion object. Like `transformToMaybe(...): UpstreamDownstreamGenericTests`

Reply:
Just realised that we also need `single-to-observable` etc. generic tests. What do you think of:
- rename everywhere `UpstreamDownstreamGenericTests` to e.g. `SingleToSingleTests`, `ObservableToObservableTests`, `SingleToMaybeTests`? Each in separate file in corresponding packages.